libmattermost.c

```diff
@@ -1884,7 +1930,9 @@ mm_role_to_purple_flag(MattermostAccount *ma, const gchar *rolelist)
 			
 		} else if (purple_strequal(role, "channel_admin")) {
 			flags |= PURPLE_CHAT_USER_OP;
-		}
+		} else if (purple_strequal(role, "system_admin")) {
+			flags |= PURPLE_CHAT_USER_FOUNDER;

```

Comment:
I wanted to have it available for the Get Info, but yes, will try not to for the future !